backend/pkg/database/common.go

```diff
@@ -88,7 +88,7 @@ func FilterIsBool(db *gorm.DB, column string, value *bool) *gorm.DB {
 }
 
 func FilterIs(db *gorm.DB, column string, values []string) *gorm.DB {
-	if len(values) == 0 {
+	if values == nil {
 		return db
 	}
 	return db.Where(fmt.Sprintf("%s IN ?", column), values)
```

Comment:
so now we will run it on an empty list and will get nothing? That basically the fix, right?

Reply:
yeah, we get `SELECT ... WHERE id in []` in the query